cric-predict/src/Components/PublicGroups.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import '../Stylesheets/App.css';
+import axios from 'axios';
+import {Button, Modal, Form, Row, Col}  from 'react-bootstrap';
+
+
+export default class PublicGroupsComponent extends React.Component  {
```

Comment:
can't we just handle this in one component? I'm assuming we've two different components with almost same rendering logic